packages/react-server/src/ReactFlightServer.js

```diff
@@ -470,7 +470,7 @@ function RequestInstance(
   this.fatalError = null;
   this.destination = null;
   this.bundlerConfig = bundlerConfig;
-  this.cache = new Map();
+  this.cache = new WeakMap();
```

Comment:
I might have missed some instances since Flow didn't complain here since we silence errors later. Though I assume we only init this once when we create the request and not reassign it later?